journal/modernized/13-chapter-11.adoc

```diff
@@ -145,28 +150,33 @@ and live and walk in that which makes for peace, and in the noble seed of life,
 in which the Father is honoured.
 And as you live, walk and persevere herein, you will see the evil seeds-man,
 and in every concern, keep in dominion over his sowings, through any occasion whatsoever,
-which is the thought and counsel of Israels God unto you.
+which is the thought and counsel of Israel`'s God unto you.
 That in and through the brightness of the everlasting, glorious Truth,
 you may shine in all your several places and services, and be good examples in love,
 fear and unity, and all sweetness of life to the adjacent country;
-that the rain of Gods heavenly love in showers may come
+that the rain of God`'s heavenly love in showers may come
 down daily upon and among you in every meeting and family;
 that you may grow like a well-watered garden, full of trees and plants of righteousness,
 and full of the divine virtue of the holy Vine of life.
 God Almighty be with you in all your meetings and families,
 causing it thus to be with you,
 is the breathing of your travailing-spirited friend and brother,
 
+[.signed-section-signature]
 Charles Marshall.
 
+[.signed-section-context-close]
 For the Quarterly Meeting in Oxford, and from there +++[+++to be]
 sent to Friends other meetings.
 
+[.signed-section-context-close]
 Cono-Alline, the 28th of Tenth Month, 1679.
 
-A General Epistle to Friends and professors of the truth,
+[.blurb]
+=== A General Epistle to Friends and professors of the truth,
 +++[+++on the necessity of the new birth.]
```

Comment:
this comment in brackets needs to be bumped up onto the same with the rest of the heading (line 176). And since the letter before this and after this had letter-headings (and not blurbs), lets use a letter-heading on this one too. I know I told you that on some of the really long ones I think it looks better with blurbs instead of letter-headings. I don't exactly know where to draw the line with that, but I kinda don't think that we should go back and forth (between letter-headings and blurbs) in the same chapter. 👍🏻 The Original will need the same done to it.

Reply:
Sounds good. I know what you mean about going back an forth between letter headings and blurbs.